### PR TITLE
[ENDPOINT] DELETE /categories/:id - OT102-99

### PR DESCRIPTION
**Pull request by joselo85**
# Description:

AS: User Administrator
I WANT: To delete an existing category
TO: Be able to keep information up to date.

## Acceptance Criteria:

Request to DELETE /categories/:id - Should validate that said category exists and delete it, otherwise it should return an error.

### DB previous to deletion:
OT102-99-Pre-delete
***
### Request to route with inexistent ID:
OT102-99-No Category was found
***
### Request to route with valid ID:
OT102-99-Success
***
### Resulting DB, deletedAt field generated:
OT102-99-Success-DB

### CODE REVIEW

File: controllers/categories.js [hunk 1]
```diff
@@ -14,4 +17,21 @@ module.exports = {
...
+      endpointResponse({
+        res,
+        msg: 'Category was succesfully deleted',
+        body: deletedCategory,
+      })
+    } catch (error) {
+      next(
+        new ErrorObject(
+          `[Error deleting Category] - [category - delete]: ${error.message}`,
+          404,
```
Comment (dvelezroman) [inline controllers/categories.js]: sure.. this 404 is the error code we want  ? maybe 500 ...

File: routes/categories.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 const express = require('express')
-const { get } = require('../controllers/categories')
+const { get, destroy } = require('../controllers/categories')
 
 const router = new express.Router()
 
+// Get Categories listing:
 router.get('/', get)
+// Delete Category by ID:
+router.delete('/:id', destroy)
```
Comment (dvelezroman) [inline routes/categories.js:9]: good

**Review by dvelezroman — APPROVED** [review]
Good PR... just some few comments

File: controllers/categories.js (hunk 1 above)
Comment (joselo85) [reply controllers/categories.js]: You're right. Will check that out! Thanks!

**Review by alexiscjscab — APPROVED** [review]

**Review by JonatanGarbuyo — APPROVED** [review]

**Pull request merged**